src/main/java/resources/PersonResources.java

```diff
@@ -55,22 +57,39 @@ public List fetchPersonsWithSlots(@QueryParam("startDate")String startDate, @Que
                     Map<String, Object> item = new HashMap<>();
                     item.put("person", person);
                     item.put("slots", slots
-                        .stream()
-                        .filter(slot -> slot.getPerson().getId() == person.getId())
-                        .map(slot -> {
-                            Map<String, Object> map = new HashMap<>();
-                            map.put("id", slot.getId());
-                            map.put("person", slot.getPerson().getId());
-                            map.put("day", slot.getSlotsDate());
-                            map.put("slot", slot.getSlot().getId());
-                            map.put("type", slot.getType().getType());
+                            .stream()
+                            .filter(slot -> slot.getPerson().getId() == person.getId())
+                            .map(slot -> {
+                                Map<String, Object> map = new HashMap<>();
+                                map.put("id", slot.getId());
+                                map.put("person", slot.getPerson().getId());
+                                map.put("day", slot.getSlotsDate());
+                                map.put("slot", slot.getSlot().getId());
+                                map.put("type", slot.getType().getType());
 
-                            return map;
-                        })
-                        .collect(Collectors.toCollection(ArrayList::new)));
+                                return map;
+                            })
+                            .collect(Collectors.toCollection(ArrayList::new)));
 
                     return item;
                 })
                 .collect(Collectors.toCollection(ArrayList::new));
     }
+
+    @GET
+    @Path("/getNote")
+    @UnitOfWork
+    public Notes getNote(@QueryParam("id") Long personId,
```

Comment:
person id should never be a query param as it is a resource identifier. It should be a path param